subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -300,6 +300,57 @@ ull_conn_iso_lll_stream_get_by_group(struct lll_conn_iso_group *cig_lll,
 	return &cis->lll;
 }
 
+struct lll_conn_iso_stream *
```

Comment:
Maybe a comment about how it's sorting? It's not obvious from the name

Reply:
Added comments.